webapp_v2/src/components/studies/StudiesList.tsx

```diff
@@ -229,6 +238,14 @@ function StudiesList(props: PropTypes) {
     setFolderList(folder.split("/"));
   }, [folder]);
 
+  const updateLastScroll = debounce(
```

Comment:
A new debounced function is created after each render and is not canceled at unmount.
Use `useDebounce` from "react-use", or create a new hook that return a debounced function (better for me).